{"uuid": "6dd92784-54e7-4645-9503-792f2bf8a7ba", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "226759c2-ed50-49ef-93c8-32612fa85d43", "event": "runner_on_unreachable", "pid": 8386, "created": "2022-04-22T13:39:52.447332", "parent_uuid": "9e73ed77-7fb2-876e-ec3d-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "37e1a10e-f6e2-4d20-a668-a20f37527b83", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-876e-ec3d-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-876e-ec3d-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:39:52.429966", "end": "2022-04-22T13:39:52.447253", "duration": 0.017287, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "6dd92784-54e7-4645-9503-792f2bf8a7ba"}}